packages/shared/styles/components/organisms/SfHero.scss

```diff
@@ -1,24 +1,31 @@
 @import "../../helpers";
 .sf-hero-item {
-  display: flex;
-  flex-direction: var(--hero-item-flex-direction, row);
-  align-items: var(--hero-item-align-items, center);
-  justify-content: var(--hero-item-justify-content, flex-start);
-  box-sizing: border-box;
-  min-height: var(--hero-item-height, 14rem);
-  width: var(--hero-item-width, 100%);
-  padding: var(--hero-item-padding, var(--spacer-sm));
+	display: flex; 
+	justify-content: var(--hero-item-justify-content, flex-start);    
+  box-sizing: border-box;  
+  width: var(--hero-item-width, 100%);  
   color: var(--hero-item-color, var(--c-text));
   @include background(
     --hero-item-background,
     var(--_hero-item-background-color, transparent),
     var(--_hero-item-background-image)
   );
-  &__container {
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    width: var(--hero-item-container-width, 50%);
+  &__wrapper {
+		display: flex;
+		flex-direction: var(--hero-item-wrapper-flex-direction, column);
+		align-items: var(--hero-item-wrapper-align-items, flex-start);
+		justify-content: var(--hero-item-wrapper-justify-content, center);
+		width: var(--hero-item-wrapper-width, 100%);	
```

Comment:
What do you think about lower value? Now it's a bit too long :) 
![Zrzut ekranu z 2020-11-12 15-21-11](https://user-images.githubusercontent.com/46591755/98951678-d80bad80-24fa-11eb-98b9-e749ef55b86f.png)

Reply:
@justyna-13 I think the wrapper should have 100% width to be fully clickable, or not?

Reply:
right :) wrapper should be fully clickable, so we just need to adjust the title width

Reply:
@justyna-13 I fixed that. Do you think it is better now?